### PR TITLE
modified topicCard.html.erb

### LINKED ISSUE #4406: 'click to unfollow' is not working in /tag section
### Please describe the problem (or idea)

> What happened just before the problem occurred? Or what problem could this idea solve?

image

here we are not able to unfollow the tags direct which we are following and it is redirected to this page,
but we should be able to unfollow directly from ```/tags```

image

### Please show us where to look

https://publiclab.org/tags

### What's your PublicLab.org username?

geekychaser

> This can help us diagnose the issue: 

### Browser, version, and operating system

> Many bugs are related to these -- please help us track it down and reproduce what you're seeing!

****

## Thank you!

Your help makes Public Lab better! We *deeply* appreciate your helping refine and improve this site. 

To learn how to write really great issues, which increases the chances they'll be resolved, see:

https://publiclab.org/wiki/developers#Contributing+for+non-coders

### PR DESCRIPTION
**Pull request by snpd25**
Fixes #4406 
The page does not redirects to the unsubscribe page now upon clicking the unfollow symbol, but it still needs to be refreshed. I am working to improve it.

### CODE REVIEW

**Comment by snpd25** [comment]
@SidharthBansal , please review this.

**Comment by SidharthBansal** [comment]
Awesome. Thanks for contributing at Public Lab

>> but it still needs to be refreshed. I am working to improve it.
Have you attached those changes in this PR?

**Comment by SidharthBansal** [comment]
Can you please provide us with a gif to demonstrate the working on localhost?

**Comment by snpd25** [comment]
@SidharthBansal 
> but it still needs to be refreshed. I am working to improve it.
Have you attached those changes in this PR?
I need help regarding this, I have tried `onclick="window.location.reload(true);"` and `redirect_back(fallback_location: root_path)` in the delete method of subscription controller. Both didn't work. Currently it is functioning like this:
Peek 2019-10-19 09-57

**Comment by jywarren** [comment]
This is awesome. Really well done!

File: app/views/tag/_topicCard.html.erb [hunk 1]
```diff
@@ -5,7 +5,7 @@
     <div class="card shadow-sm p-2 mb-3 bg-white rounded" >
       <div class="card-header mt-2" style=" padding:0.8em; background-color: inherit; border:none;">
         <% if current_user && current_user.following(tag.name) && controller.controller_name != 'tag' %>
-          <a style="margin-top:-8px;" data-method="delete" data-confirm="Are you sure you'd like to unfollow this topic?" rel="tooltip" title="<%= t('tag.show.unfollow') %>" class="pull-right" href="/unsubscribe/tag/<%= tag.name %>"><i style="color:#ccc;" class="fa fa-times"></i></a>
+          <a style="margin-top:-8px;" data-method="delete" data-confirm="Are you sure you'd like to unfollow this topic?" rel="tooltip" title="<%= t('tag.show.unfollow') %>" class="pull-right" href="/unsubscribe/tag/<%= tag.name %>" data-remote="true" onclick="window.location.reload(true);"><i style="color:#ccc;" class="fa fa-times"></i></a>
```
Comment (jywarren) [inline app/views/tag/_topicCard.html.erb]: ah, clever! Does this work both if it succeeds or fails?

Comment (snpd25) [reply app/views/tag/_topicCard.html.erb]: I tried this, but it doesn't work. The page gets reloaded without toggling unfollow to follow.

Comment (jywarren) [reply app/views/tag/_topicCard.html.erb]: Ah, sorry, so this PR is not quite ready then? Would you like to try a different approach to this? Perhaps we have to add a `return_to` parameter to the subscription controller, and make it possible to redirect back?https://github.com/publiclab/plots2/blob/master/app/controllers/subscription_controller.rb#L27-L81

**Comment by SidharthBansal** [comment]
As the person is inactive for more than a month, I am closing the PR. In case you want to push changes please feel free to open a new PR OR reopen this PR and add additional changes to it.
Thanks for contributing on Public Lab

**Pull request closed without merging**